modules/tribes/client/components/CirclePage.component.js

```diff
@@ -0,0 +1,190 @@
+            <h2>{t('This circle is not here...')}</h2>
+            <p className="lead">
+              {t(
+                `The circle you're seeking isn't here. Check if your address is correct.`,
+              )}
+            </p>
+            <a onClick={goBack} className="btn btn-primary">
+              {t('See other circles')}
+            </a>
+          </div>
+        </div>
+      </section>
+    );
+  }
+
+  // Circle found
+  return (
+    <section
+      className={`board tribe-image tribe-header ${!user ? 'is-guest' : ''}`}
+      tr-tribe-styles="{{::tribeCtrl.tribe}}"
+      tr-tribe-styles-dimensions="1400x900"
+    >
+      <div className="tribe-header-info">
+        <div className="container">
+          <div className="row no-gutters">
+            <div className="col-xs-12">
+              <a
+                onClick={goBack}
+                className="btn btn-lg btn-link tribe-header-back"
+              >
+                <i className="icon-left"></i>
+                {t('More circles')}
+              </a>
+            </div>
+          </div>
+          <div className="row">
+            {/* For authenticated members */}
+            {user && (
+              <div
+                className="col-xs-10 col-sm-offset-1 col-sm-7 col-md-6 col-lg-5"
+                // ng-if="app.user"
+              >
+                <p className="lead tribe-pre">{t('Circle')}</p>
+                <h2 className="font-brand-regular tribe-title">
+                  {circle.label}
+                </h2>
+                <div className="tribe-meta">
+                  {circle.count === 0 && t('No members yet')}
+                  {circle.count === 1 && t('One memeber')}
+                  {circle.count > 1 && t(`${circle.count} members`)}
+                </div>
+                {circle.description && (
+                  <div className="lead tribe-meta">{t(circle.description)}</div>
+                )}
+                <br />
+                <br />
+                <JoinButton tribe={circle} user={user} onUpdated={() => {}} />
+                &nbsp;
+                <a
+                  className="btn btn-lg btn-default"
+                  href={`/search?tribe=${circle.slug}`}
+                  type="button"
+                >
+                  {t('Find members')}
+                </a>
+                <br />
+                <br />
+                <p className="lead tribe-intro">
+                  {t(
+                    'Trustroots is built on communities. Share this page within your community and invite them to join!',
+                  )}
+                </p>
+                {circle.label && (
+                  <>
+                    {/* Twitter share button */}
+                    <div
+                      className="tribe-share"
+                      tr-share-twitter
+                      data-text={t(
+                        `Join to meet, host and get hosted by Trustroots circle ${circle.label} — I'm in!`,
```

Comment:
We can use [i18n-next interpolation](https://www.i18next.com/translation-function/interpolation) here instead:
```js
t("Join to meet, host and get hosted by Trustroots circle {{circle}} — I'm in!", { circle: circle.label })
```